MetabolicAtlas/data-files
# feat: add new dataset for map overlay

This branch will:
- Update instructions on how to add new data overlay datasets
- Add a new data overlay dataset for Yeast GEM

To test:
- Read documentation and make sure it makes sense
- Use dataset in MetAtlas by `build-stack && start-stack`
